auth-server/README.md

```diff
@@ -0,0 +1,17 @@
+# Pet Clinic Authentication Server
+
+It is a server which provides authentication and authorization services.
+
+## Initialize database structure
+1. Use Gradle to run by `gradlew -p auth-server createDatabase` in the root sub-project folder. It creates user and database.
+(see 'gradle.properties' file for configuration)
```

Comment:
it is possible to link this name to file content (add a link which can be used to open such file)